### PR TITLE
Jupyter 10.0

### PR DESCRIPTION
**Pull request by epifanio**
Update jupyter notebook and ipywidgets to support and enable latest JS widget. + adding sql geoalchemy2 to easy access postgis DB + addding ipyleaflet widget.

### CODE REVIEW

File: bin/install_jupyter.sh [hunk 1]
```diff
@@ -74,5 +88,18 @@ cp "$BUILD_DIR"/../app-data/jupyter/cartopy_simple.ipynb \
    "$USER_HOME/jupyter/notebooks/projects/CARTOPY/"
 cp -r /home/user/jupyter /etc/skel
 
+# add geoalchemy to easy access to postgis from python
+wget -c --tries=3 --progress=dot:mega \
+    "http://download.osgeo.org/livedvd/data/jupyter/debs/python-geoalchemy2_0.3.0_all.deb"
+dpkg -i
```
Comment (kalxas) [inline bin/install_jupyter.sh]: deb file missing

**Comment by kalxas** [comment]
I would prefer to have those new packages included in Launchpad... I will look into them

**Comment by epifanio** [comment]
the files are here:

http://download.osgeo.org/livedvd/data/jupyter/debs/

http://download.osgeo.org/livedvd/data/jupyter/debs/python-geoalchemy2_0.3.0_all.deb

perhaps typo in the script?

Ok, Ping me back when you have those debs in launchpad and enabled in a new build so that I can make the notebook for quickstart on top of the new iso.

File: bin/install_jupyter.sh (hunk 1 above)
Comment (epifanio) [inline bin/install_jupyter.sh]: copy-paste from the script: 

http://download.osgeo.org/livedvd/data/jupyter/debs/python-geoalchemy2_0.3.0_all.deb

copy paste from "http://download.osgeo.org/livedvd/data/jupyter/debs/":

http://download.osgeo.org/livedvd/data/jupyter/debs/python-geoalchemy2_0.3.0_all.deb

the file seems to be there ..

Comment (epifanio) [inline bin/install_jupyter.sh]: is the "dpkg -i" instruction ... which has a missed package name .. updating it

**Comment by kalxas** [comment]
@epifanio new nightly including your packages:
http://aiolos.survey.ntua.gr/gisvm/dev/osgeo-live-nightly-build194-amd64-3c0714c.iso
Can you please confirm it is ok?

**Comment by kalxas** [comment]
@epifanio Notebook 4.2.1 has landed in our PPA last night. I tested nightly 191 and it works fine. I will continue with the rest of the required packages.

**Comment by kalxas** [comment]
Also now in the PPA: widgetsnbextension, ipywidgets 5.1.5, ipyleaflet

**Comment by kalxas** [comment]
@epifanio is geoalchemy necessary for the demo?

**Comment by kalxas** [comment]
merged

**Pull request closed without merging**